Src/FluentAssertions/FluentAction.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Threading.Tasks;
+using JetBrains.Annotations;
+
+namespace FluentAssertions
+{
+    /// <summary>
+    /// Contains static methods to help with exception assertions on actions.
+    /// </summary>
+    [DebuggerNonUserCode]
+    public static class FluentAction
+    {
+        /// <summary>
+        /// Invokes the specified action so that you can chain it with ShouldThrow, etc.
+        /// </summary>
+        [Pure]
+        public static Action Invoking(Action action) => action;
+
+        /// <summary>
+        /// Invokes the specified action so that you can chain it with ShouldThrow, etc.
+        /// </summary>
+        [Pure]
+        public static Func<T> Invoking<T>(Func<T> func) => func;
```

Comment:
@dennisdoomen If we decide on the name `Invoking` for `Fun<T>` here, then #872 should also be named `Invoking` for consistency.